ssl.go

```diff
@@ -1,195 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"io/ioutil"
-	"os"
-	"strings"
-	"time"
-
-	"github.com/remind101/emp/Godeps/_workspace/src/github.com/bgentry/heroku-go"
-)
-
-var cmdSSL = &Command{
```

Comment:
Hmm - I don't suppose there's a way to keep them, but not include them in help?  That way if someone tries to use them out of habit, we can at least respond with a `This is deprecated` error, but folks don't see them and attempt to use them just from the help.